spring-cloud-gcp-samples/pom.xml

```diff
@@ -46,6 +46,7 @@
 		<maven.javadoc.skip>true</maven.javadoc.skip>
 		<maven.sources.skip>true</maven.sources.skip>
 		<sonar.skip>true</sonar.skip>
+		<cloud-spanner-r2dbc.version>1.2.2</cloud-spanner-r2dbc.version>
```

Comment:
The version should be added to the BOM `spring-cloud-gcp-dependencies/pom.xml`.

Reply:
sure, moved it to `spring-cloud-gcp-dependencies/pom.xml`